src/openms/source/METADATA/PeptideHit.cpp

```diff
@@ -85,10 +85,10 @@ namespace OpenMS
   PeptideHit::PeptideHit(PeptideHit&& source) noexcept :
     MetaInfoInterface(std::move(source)), // NOTE: rhs itself is an lvalue
     sequence_(std::move(source.sequence_)),
-    score_(std::move(source.score_)),
-    analysis_results_(std::move(source.analysis_results_)),
-    rank_(std::move(source.rank_)),
-    charge_(std::move(source.charge_)),
+    score_(source.score_),
+    analysis_results_(source.analysis_results_),
```

Comment:
why no move here?

Reply:
Discouraged since it's a built-in type.

Reply:
ah yeah right

Reply:
hmm I understand but also it seems more readable to use `std::move` and it could lead to problems if we decide to change the type later on and then there is no move...

Reply:
readded the move.